ui/static/ui/js/static_assets.jsx

```diff
@@ -10,49 +17,49 @@ define('static_assets', [
         results: []
       };
     },
-    componentDidMount: function () {
-      var thiz = this;
-
-      Utils.getCollection("/api/v1/repositories/" + this.props.repoSlug +
-          "/learning_resources/" +
-          this.props.learningResourceId + "/static_assets/")
-          .done(function (assets) {
-            if (!thiz.isMounted()) {
-              // In time AJAX call happens component may become unmounted
-              return;
-            }
-
-            thiz.setState({
-              results: assets
-            });
-          })
-          .fail(function () {
-            thiz.setState({
-              message: {
-                error: "Unable to read information about static assets."
-              }
-            });
-          });
+    makeElement: function(asset) {
+      return <li className="list-group-item" key={asset.asset}
+                 style={{height: elementHeight}}>
+        <a href={asset.asset}>{asset.name}</a>
+      </li>;
     },
     render: function () {
-      var staticAssets = _.map(this.state.results, function (asset) {
-        return <li className="list-group-item" key={asset.asset}>
-            <a href={asset.asset}>{asset.name}</a>
-          </li>;
-      });
+      var url = "/api/v1/repositories/" + this.props.repoSlug +
+          "/learning_resources/" +
+          this.props.learningResourceId + "/static_assets/";
+
+      // Infinite requires you to specify a containerHeight. This value
+      // is a guess at how big the area should be relative to the height.
+      var containerHeight = $(window).height() - 200;
```

Comment:
I was trying to pick a height which is the window height minus 200 pixels for the panel borders. It's also just a guess